system/Database/SQLSRV/Connection.php

```diff
@@ -229,9 +229,7 @@ protected function _indexData(string $table): array
             $obj->name = $row->index_name;
 
             $_fields     = explode(',', trim($row->index_keys));
-            $obj->fields = array_map(static function ($v) {
-                return trim($v);
-            }, $_fields);
+            $obj->fields = array_map(static fn ($v) => trim($v), $_fields);
```

Comment:
Not related to PHP 7.4 migration but can be simplified further.
```suggestion
            $obj->fields = array_map('trim', $_fields);
```